### DISCUSSION TITLE
是否有方法可以直接在PPStructure将训练后的版面分析的模型调用

### DISCUSSION THREAD

**Discussion by 77bw** [PaddlePaddle/PaddleOCR#13237] [Q&A] [answered]
官方的推理是直接在PaddleDetection上；
python3 deploy/python/infer.py \
    --model_dir=output_inference/picodet_lcnet_x1_0_layout/ \
    --image_file=docs/images/layout.jpg \
    --device=CPU
    
 但是我想在paddleocr的版面分析上使用PaddleDetection训练的模型，该如何用呢？我尝试过将layout_model_dir自定义为本地训练模型，但是该方法不可行
 import os
import cv2
from paddleocr import PPStructure,save_structure_res

table_engine = PPStructure(table=False, ocr=False, show_log=True)

save_folder = './output'
img_path = 'ppstructure/docs/table/1.png'
img = cv2.imread(img_path)
result = table_engine(img)
save_structure_res(result, save_folder, os.path.basename(img_path).split('.')[0])

for line in result:
    line.pop('img')
    print(line)

**Comment by SWHL** [comment, marked as answer]
可以使用我整理的[RapidLayout](https://github.com/RapidAI/RapidLayout)

**Reply by 77bw** [reply]
好的谢谢兄弟
